openshift/kustomize/api/base/route-editor.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   namespace: default
   annotations:
     haproxy.router.openshift.io/rewrite-target: /
+    haproxy.router.openshift.io/balance: source
+    haproxy.router.openshift.io/cookie_name: mmi
```

Comment:
This is supposed to use a cookie for the session, but I can't see it in the client